### ISSUE TITLE
[Regression] Some RPCS3 crashes (e.g. installing firmware after a game is started) on Qt lib (#13876)

### ISSUE THREAD

**Issue by digant73** [RPCS3/rpcs3#16455]
### Quick summary

The issue is present since PR #13876, build 0.0.34-17155. Made several tests often reproducing the issue.
I saw the issue starting a game with firmware not yet installed. Trying to install the fw (when asked) often causes a crash on Qt lib.
Due to the bugged PR, the issue is probably more generic (not only related to fw installation).
Below an example of the issue:

![image](https://github.com/user-attachments/assets/51b478fc-4f86-4d41-b869-45749be695a8)

### Build with regression

0.0.34-17155

### Attach two log files

[RPCS3.log](https://github.com/user-attachments/files/18240178/RPCS3.log)

**Comment by AniLeo** [comment]
Tried 5x on Linux, could not reproduce
Need Information: New repro on latest build

**Comment by AniLeo** [comment]
@digant73 can you still reproduce this?

**Comment by digant73** [comment]
yes, the issue is still there but slightly different. I didn't see a crash of the emulator. Now after the fw is installed the game is launched but the windows (reporting Compiling PPU etc.) is immediately closed. I have to double-click again on the game and then it seems to properly start (it compiles PPU etc.).

**Comment by AniLeo** [comment]
When you're missing firmware, it installs and compiles the firmware. Then you need to launch the game manually afterwards. If you were initiating a game boot without firmware, installing firmware cancels that as you can only boot one thing at a time. I'm not sure what's the expected behaviour you need here.

**Comment by digant73** [comment]
if so and the crash seems now no more present I think this bug report can be closed. There is still the bug discussed in #16492, After 3 times you open the dialog to select the fw the entire emulator freezes

**Comment by AniLeo** [comment]
As it's still an issue can you open a new ticket for https://github.com/RPCS3/rpcs3/pull/16492#issuecomment-2568371432 then?

**Comment by digant73** [comment]
yes I will, just to keep track of that residual minor bug
